packages/stats/gbstats/bayesian/tests.py

```diff
@@ -313,3 +351,121 @@ def compute_result(self) -> BayesianTestResult:
                 result, self.traffic_proportion_b, self.phase_length_days
             )
         return result
+
+
+class GaussianEffectABTest(GaussianBayesianABTest):
+    def __init__(
+        self,
+        stat_a: Union[
+            SampleMeanStatistic,
+            RatioStatistic,
+            QuantileStatistic,
+            QuantileStatisticClustered,
+        ],
+        stat_b: Union[
+            SampleMeanStatistic,
+            RatioStatistic,
+            QuantileStatistic,
+            QuantileStatisticClustered,
+        ],
+        config: Union[EffectConfigFlatPrior, EffectConfigStandardPrior],
```

Comment:
This is what i'm referring to above, this should just be `EffectBayesianConfig`.

Reply:
fixed